specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -327,105 +256,32 @@ in the request body.
-  @get
-  GetTestRunFile is Azure.Core.Foundations.Operation<
-    {
-      @doc("""
-Unique name for the load test run, must contain only lower-case alphabetic,
-numeric, underscore or hyphen characters.
-""")
-      @path
-      testRunId: string;
-
-      @doc("Test run file name with file extension")
-      @path
-      fileName: string;
-    },
-    FileInfo
-  >;
+  deleteTestRun is StandardResourceOperations.ResourceDelete<TestRun>;
 
   @summary("Get all test runs with given filters")
   @doc("Get all test runs with given filters")
-  @route("/test-runs")
-  @get
-  ListTestRuns is Azure.Core.Foundations.Operation<
-    {
-      @doc("""
-Sort on the supported fields in (field asc/desc) format. eg: executedDateTime
-asc. Supported fields - executedDateTime
-""")
-      @query
-      orderby?: string;
-
-      @doc("""
-Prefix based, case sensitive search on searchable fields - description,
-executedUser. For example, to search for a test run, with description 500 VUs,
-the search parameter can be 500.
-""")
-      @query
-      search?: string;
-
-      @doc("Unique name of an existing load test.")
-      @query
-      testId?: string;
-
-      @doc("Start DateTime(ISO 8601 literal format) of test-run execution time filter range.")
-      @query
-      executionFrom?: offsetDateTime;
-
-      @doc("End DateTime(ISO 8601 literal format) of test-run execution time filter range.")
-      @query
-      executionTo?: offsetDateTime;
-
-      @doc("Comma separated list of test run status.")
-      @query
-      status?: string;
-
-      @doc("Number of results in response.")
-      @query
-      maxpagesize?: int32;
-    },
-    TestRunsList
+  listTestRuns is StandardResourceOperations.ResourceList<
+    TestRun,
+    ListQueryParametersTrait<ListTestRunQueryParams>
   >;
 
+  @summary("Get test run file by file name.")
+  @doc("Get test run file by file name.")
+  getTestRunFile is StandardResourceOperations.ResourceRead<TestRunFileInfo>;
+
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id"
   @summary("Stop test run by name.")
   @doc("Stop test run by name.")
-  @route("/test-runs/{testRunId}:stop")
-  @post
-  StopTestRun is Azure.Core.Foundations.Operation<
-    {
-      @doc("""
-Unique name for the load test run, must contain only lower-case alphabetic,
-numeric, underscore or hyphen characters.
-""")
-      @path
-      testRunId: string;
-    },
-    TestRun
-  >;
+  @example("./examples/StopTestRun.json", "StopTestRun")
+  @operationId("StopTestRun")
+  stop is StandardResourceOperations.ResourceAction<TestRun, {}, TestRun, {}>;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
   @summary("List the metric namespaces for a load test run.")
   @doc("List the metric namespaces for a load test run.")
   @route("/test-runs/{testRunId}/metric-namespaces")
   @get
-  ListMetricNamespaces is Azure.Core.Foundations.Operation<
+  listMetricNamespaces is Azure.Core.Foundations.Operation<
```

Comment:
LoadTestRun_ListMetricNamespaces this is the operation ID in our existing GA spec. Should we change it when we move to TypeSpec? It will reflect in our existing spec as well.